CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ All notable changes to the Zowe Client Python SDK will be documented in this fil
 
 ### Enhancements
 
-- Added logger class to core SDK [#185](https://github.com/zowe/zowe-client-python-sdk/issues/185)
+- Refactor Files class into proper classes [#264](https://github.com/zowe/zowe-client-python-sdk/issues/264)
```

Comment:
Should we list the new classes (e.g. Datasets, USSFiles, FileSystems) as the enhancements?

Maybe something like:
```suggestion
- Refactored Files class into `Datasets`, `USSFiles`, and `FileSystems`. [#264](https://github.com/zowe/zowe-client-python-sdk/issues/264)
```

Or something like this:
```suggestion
- Added classes for handling `Datasets`, `USSFiles`, and `FileSystems` in favor of the single Files class. [#264](https://github.com/zowe/zowe-client-python-sdk/issues/264)
```